java/src/jmri/web/server/WebServer.java

```diff
@@ -143,6 +144,7 @@ public void start() {
     @Override
     public void stop() throws Exception {
         server.stop();
+        zeroConfService.stop();
```

Comment:
I did not catch this in the original PR, but this should not be here. It needs to be in the method ```lifeCycleStopped()``` (the same way as in ```lifeCycleStopping()```). That way, if the Jetty server is stopped, the zeroConfService is no longer advertised. (If that is failing, then it indicates the error is that this class is not listening to the Jetty server correctly, and that needs to be fixed.

Reply:
I went ahead and moved that.  

That was added to make the start/stop test not cause an issue further down the test sequence because the service was still advertised.  

The service advertisement starts in the start routine, and if we stop before jetty is fully initialized, then we might still have the advertisement hanging out there.  The start/stop test was already merged in, so we'll see if there are issues with this in lifeCycleStopped().